pom.xml

```diff
@@ -51,6 +53,10 @@
         <enforceCorrectDependencies>true</enforceCorrectDependencies>
         <fasterxmlVersion>2.9.8</fasterxmlVersion>
         <springDataVersion>2.0.9.RELEASE</springDataVersion>
+        <caf.worker-batch-framework.version>3.2.0-SNAPSHOT</caf.worker-batch-framework.version>
```

Comment:
You should not be using the latest SNAPSHOT version unless there is a feature in it that you need.  If the latest release, `3.1.0-161` in this case, is sufficient then that should be used so that we know when it comes to releasing what we need to release.